File: test/fixtures/node-superagent/get_with_data2.js
import request from 'superagent';

request
  .put('http://localhost:28139/v2/alerts_policy_channels.json')
  .query({
    'policy_id': 'policy_id',
    'channel_ids': 'channel_id'
  })
  .set('X-Api-Key', '{admin_api_key}')
  .type('json')
  .then(res => {
     console.log(res.body);
  });

File: test/fixtures/r-httr2/get_with_data2.r
library(httr2)

request("http://localhost:28139/v2/alerts_policy_channels.json") |>
  req_method("PUT") |>
  req_url_query(
    policy_id = "policy_id",
    channel_ids = "channel_id"
  ) |>
  req_headers(`X-Api-Key` = "{admin_api_key}") |>
  req_perform()
